<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>761</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Dubai</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>80</y>
      <width>391</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <widget class="QLabel" name="searchingLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>NULL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Found Food Place</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Searching for Food Place</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="foundFoodLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>NULL</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>STATUS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Current Time</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>570</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Meal Times Setting</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>570</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>851</width>
      <height>340</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QDial" name="meal2HoursKnob">
       <property name="maximum">
        <number>23</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QDial" name="meal3HoursKnob">
       <property name="maximum">
        <number>23</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Dubai</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Meal 1 Time:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="meal1AdjustLabel">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="setMeal3Btn">
       <property name="text">
        <string>Set Meal 3 Time</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Meal 3 Time:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="setMeal1Btn">
       <property name="text">
        <string>Set Meal 1 Time</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="setMeal2Btn">
       <property name="text">
        <string>Set Meal 2 Time</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDial" name="meal3MinsKnob">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Hours Knob</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Meal 2 Time:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDial" name="meal1HoursKnob">
       <property name="maximum">
        <number>23</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDial" name="meal1MinsKnob">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="meal3AdjustLabel">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="meal2AdjustLabel">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDial" name="meal2MinsKnob">
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Minutes Knob</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>480</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLabel" name="meal2TimeLabel">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Dubai</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Meal 1 Time:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Meal 2 Time:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Meal 3 Time:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="meal1TimeLabel">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="meal3TimeLabel">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="StatusLabel_4">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>450</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Meal Times</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>20</width>
      <height>581</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>440</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>851</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>-190</x>
      <y>220</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>20</width>
      <height>301</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>80</y>
      <width>20</width>
      <height>341</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_16">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>20</width>
      <height>341</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_17">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>80</y>
      <width>20</width>
      <height>341</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_18">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>540</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_19">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>610</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_20">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>510</y>
      <width>21</width>
      <height>161</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_21">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>470</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>440</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>440</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>680</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>430</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_22">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>80</y>
      <width>20</width>
      <height>341</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_23">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>80</y>
      <width>20</width>
      <height>341</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1347</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
